packages/api/src/APIClient.ts

```diff
@@ -284,6 +301,76 @@ export default class APIClient {
 			),
 	}
 
+	public readonly otherIncomes = {
+		/**
+		 * Get list of other incomes
+		 */
+		list: (
+			accountId: string,
+			queryBuilders?: QueryBuilderType[]
+		): Promise<Result<{ otherIncomes: OtherIncome[]; pages: Pagination }>> =>
+			this.call(
+				'GET',
+				`/accounting/account/${accountId}/other_incomes/other_incomes${joinQueries(queryBuilders)}`,
+				{
+					transformResponse: transformListOtherIncomesResponse,
+				},
+				null,
+				'List OtherIncomes'
+			),
+		/**
+		 * Get single other income
+		 */
+		single: (
+			accountId: string,
+			otherIncomeId: string,
+			queryBuilders?: QueryBuilderType[]
+		): Promise<Result<OtherIncome>> =>
+			this.call(
+				'GET',
+				`/accounting/account/${accountId}/other_incomes/other_incomes/${otherIncomeId}${joinQueries(queryBuilders)}`,
+				{
+					transformResponse: transformOtherIncomeResponse,
+				},
+				null,
+				'Get OtherIncome'
+			),
+		/**
+		 * Post other income
+		 */
+		create: (otherIncome: OtherIncome, accountId: string): Promise<Result<OtherIncome>> => {
+			return this.call(
+				'POST',
+				`/accounting/account/${accountId}/other_incomes/other_incomes`,
+				{
+					transformResponse: transformOtherIncomeResponse,
+					transformRequest: transformOtherIncomeRequest,
+				},
+				otherIncome,
+				'Create OtherIncome'
+			)
+		},
+		update: (accountId: string, otherIncomeId: string, data: any): Promise<Result<OtherIncome>> =>
```

Comment:
should be `OtherIncom` type instead of `data: any`?

Reply:
for the update is not the whole object but just part of it. and based on the rest of the code I saw that it was always used and so I used that